plugins/coverbrowser.koplugin/covermenu.lua

```diff
@@ -0,0 +1,383 @@
+                            ["ignore_meta"] = not bookinfo.ignore_meta and 'Y' or false,
+                        })
+                        UIManager:close(self.file_dialog)
+                        self:updateItems()
+                    end,
+                },
+            })
+            table.insert(orig_buttons, {
+                { -- Allow a new extraction (multiple interruptions, book replaced)...
+                    text = _("Refresh cached book information"),
+                    enabled = bookinfo and true or false,
+                    callback = function()
+                        BookInfoManager:deleteBookInfo(file)
+                        UIManager:close(self.file_dialog)
+                        self:updateItems()
+                    end,
+                },
+            })
+
+            -- Create the new ButtonDialogTitle, and let UIManager show it
+            local ButtonDialogTitle = require("ui/widget/buttondialogtitle")
+            self.file_dialog = ButtonDialogTitle:new{
+                title = orig_title,
+                title_align = orig_title_align,
+                buttons = orig_buttons,
+            }
+            UIManager:show(self.file_dialog)
+            return true
+        end
+
+        -- Remember our function
+        self.onFileHold_ours = self.onFileHold
+    end
+
+end
+
+-- Similar to onFileHold setup just above, but for History,
+-- which is plugged in main.lua _FileManagerHistory_updateItemTable()
+function CoverMenu:onHistoryMenuHold(item)
+    -- Call original function: it will create a ButtonDialog
+    -- and store it as self.histfile_dialog, and UIManager:show() it.
+    self.onMenuHold_orig(self, item)
+    local file = item.file
+
+    local bookinfo = BookInfoManager:getBookInfo(file)
+    if not bookinfo then
+        -- If no bookinfo (yet) about this file, let the original dialog be
+        return true
+    end
+
+    -- Remember some of this original ButtonDialog properties
+    local orig_buttons = self.histfile_dialog.buttons
+    -- Close original ButtonDialog (it has not yet been painted
+    -- on screen, so we won't see it)
+    UIManager:close(self.histfile_dialog)
+
+    -- Add some new buttons to orignal buttons set
+    table.insert(orig_buttons, {
+        { -- Allow user to ignore some offending cover image
+            text = bookinfo.ignore_cover and _("Unignore file cover") or _("Ignore this cover"),
+            enabled = bookinfo.has_cover and true or false,
+            callback = function()
+                BookInfoManager:setBookInfoProperties(file, {
+                    ["ignore_cover"] = not bookinfo.ignore_cover and 'Y' or false,
+                })
+                UIManager:close(self.histfile_dialog)
+                self:updateItems()
+            end,
+        },
+        { -- Allow user to ignore some bad metadata (filename will be used instead)
+            text = bookinfo.ignore_meta and _("Unignore file metadata") or _("Ignore these metadata"),
```

Comment:
Same for `Ignore metadata`. In fact "Ignore these metadata` has an odd ring to it.